scene/gui/rich_text_label.cpp

```diff
@@ -5480,7 +5480,8 @@ void RichTextLabel::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("add_image", "image", "width", "height", "color", "inline_align", "region"), &RichTextLabel::add_image, DEFVAL(0), DEFVAL(0), DEFVAL(Color(1.0, 1.0, 1.0)), DEFVAL(INLINE_ALIGNMENT_CENTER), DEFVAL(Rect2(0, 0, 0, 0)));
 	ClassDB::bind_method(D_METHOD("newline"), &RichTextLabel::add_newline);
 	ClassDB::bind_method(D_METHOD("remove_paragraph", "paragraph"), &RichTextLabel::remove_paragraph);
-	ClassDB::bind_method(D_METHOD("push_font", "font", "font_size"), &RichTextLabel::push_font);
+	ClassDB::bind_method(D_METHOD("push_font", "font", "font_size"), &RichTextLabel::push_font, DEFVAL(0));
+	ClassDB::bind_compatibility_method(D_METHOD("push_font", "font", "font_size"), &RichTextLabel::push_font);
```

Comment:
These compatibility methods probably aren't necessary, adding a default value should not break compatibility, so it should be enough to add the error messages to `extension_api_validation` to suppress them.

Reply:
I agree your opinion, but if I have not add these compatibility methods, this pr can't pass check.

I'm not sure about the details about the check system, maybe it can be optimized?

Reply:
Currently, you can add a list of expected validation errors with the comment to this file - https://github.com/godotengine/godot/blob/master/misc/extension_api_validation/4.0-stable.expected, and it will pass the checks.

Reply:
Thanks for your teaching!